core/networkserver/adr.go

```diff
@@ -332,6 +332,34 @@ func getAdrReqPayloads(dev *device.Device, frequencyPlan *band.FrequencyPlan, dr
 				}
 			}
 		}
+	case pb_lorawan.FrequencyPlan_RU_864_870.String():
+		if dev.ADR.Failed > 0 && powerIdx > 5 {
```

Comment:
This check was added to deal with EU868 devices that didn't support pwerIdx 6 and 7. Considering that `RU_864_870` already had powerIdx 0 - 7 support from the moment it was defined, I think we can leave this check out.

Reply:
OK!